src/dvclive/live.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         resume: bool = False,
         report: Optional[str] = "auto",
         save_dvc_exp: bool = False,
-        dvcyaml: bool = True,
+        dvcyaml: Union[str, bool] = True,
```

Comment:
Changed `dvcyaml` to accept a path so any `dvc.yaml` path can be specified, but defaults to the DVC repo root.

Reply:
Why not drop the `bool` and make a default string?

Reply:
How would you turn it off? https://github.com/iterative/example-get-started for example doesn't use it and adding it would create duplicate plots.

Reply:
Optional[str] = "dvc.yaml"

And you pass dvcyaml=None.

Anyhow, it is a minor thing, no strong opinion

Reply:
It would be slightly more "breaking" but is slightly cleaner 🤔 . No strong opinion from me either.

Reply:
Let's go with this for now, not major